api/routes/course.js

```diff
@@ -182,4 +184,18 @@ router.get('/', courseController.getCoursesByAttribute);
  */
 router.get('/carousel/', courseController.getCoursesForCarousel);
 
+router.post(
```

Comment:
Pls write swagger documentation on this route.

Reply:
Done